CLAUDE.md

```
@@ -0,0 +1,802 @@
+
+# Build installer YAML (generates dist/install.yaml)
+make build-installer IMG=<registry>/osac-operator:tag
+```
+
+## Architecture
+
+### Dual-Controller Pattern
+
+Each resource has two controllers with distinct responsibilities:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)